packages/fury-adapter-swagger/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Fury Swagger Parser Changelog
 
+## 0.28.3 (2020-02-17)
+
+### Enhancements
+
+- Adds support for the `termsOfService` schema extension within the info section. 
+
+  For example, given following array schema:
```

Comment:
Array schema? I don’t think this is related to the below example.

Reply:
indeed, going to change